packages/composites/defi-pulse/src/index.ts

```diff
@@ -2,4 +2,6 @@ import { expose } from '@chainlink/ea-bootstrap'
 import { makeExecute } from './adapter'
 import { NAME, makeConfig } from './config'
 
-export = { makeExecute, makeConfig, ...expose(NAME, makeExecute()) }
+const adapterContext = { name: NAME }
+
+export = { makeExecute, makeConfig, ...expose(adapterContext, makeExecute()) }
```

Comment:
Since we are changing this line, could we modify this to be standard with a standard export style?
```suggestion
const { server } = expose(adapterContext, makeExecute())
export { NAME, makeConfig, makeExecute, server }
```